### PR TITLE
More random doctests

### PR DESCRIPTION
**Pull request by kshyatt**
Doctests passed locally.

### CODE REVIEW

File: base/docs/helpdb/Base.jl [hunk 1]
```diff
@@ -1103,6 +1146,17 @@ false
     bswap(n)
 
 Byte-swap an integer.
+
+```jldoctest
+julia> bswap(4)
```
Comment (KristofferC) [inline base/docs/helpdb/Base.jl]: Perhaps:

```
julia> a = bswap(4)
288230376151711744

julia> bswap(a)
4
```

to show the back and forth.

**Review by KristofferC — APPROVED** [review]
LGTM, just one suggestion for `bswap`

File: base/docs/helpdb/Base.jl [hunk 2]
```diff
@@ -1103,6 +1146,17 @@ false
     bswap(n)
 
 Byte-swap an integer.
```
Comment (tkelman) [inline base/docs/helpdb/Base.jl]: would be worth elaborating what this means in a few more words

**Comment by kshyatt** [comment]
Better now?

**Pull request merged**